uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/WorkDay.php

```diff
@@ -124,4 +129,13 @@ private static function validateWorkDayType($params) {
       );
     }
   }
+
+  private static function calculateNumberOfHours($params)
+  {
+    $timeFromInHours = strtotime($params['time_from']) / 3600;
```

Comment:
since **time_from** and **time_to** are not required fields .. you should check for their existence first with !empty().